<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>328</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>613</width>
    <height>328</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>613</width>
    <height>328</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>261</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/resources/tb/tb.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tw_login_register">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>361</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_login">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QLabel" name="lb_zh">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_zh">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>13351075356</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_passwd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_passwd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1234</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_commit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_register">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QPushButton" name="pb_clear__reg">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_zh_reg">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_passwd_reg">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;不多于20位密码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_zh_reg">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入11位手机号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_commit__reg">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_passwd_reg">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_confirm_reg">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_confirm_reg">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_name_reg">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;不超过10位&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_name__reg">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_zh_reg</tabstop>
  <tabstop>le_passwd_reg</tabstop>
  <tabstop>le_confirm_reg</tabstop>
  <tabstop>le_name_reg</tabstop>
  <tabstop>pb_clear__reg</tabstop>
  <tabstop>pb_commit__reg</tabstop>
  <tabstop>tw_login_register</tabstop>
  <tabstop>le_zh</tabstop>
  <tabstop>le_passwd</tabstop>
  <tabstop>pb_commit</tabstop>
  <tabstop>pb_clear</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
